### DISCUSSION TITLE
Accessibility Guidelines

### DISCUSSION THREAD

**Discussion by tonge495** [twilio-labs/paste#3338] [Q&A] [answered]
Hi Paste team,

My team is interested in conducting light accessibility audits for any new systems that we are implementing in the future.  Were there any accessibility guidelines/checklists/resources that you would recommend us to follow as a baseline?  Also, are there any automated accessibility tools that you would recommend?  I am currently using WAVE and Google Lighthouse but not sure if there were others that are preferred.

**Comment by SiTaggart** [comment, marked as answer]
Hi @tonge495 

As unexciting as it sounds, honestly the _best_ guidelines you can read are the official Web Content Accessibility Guidelines. There really isn't any getting around it. They are the definitive guidelines that are followed for things like the ADA and Section 508, and they are current. 

A new version just went into recommendation today, so I would start there to be a little ahead of the curve. https://www.w3.org/TR/WCAG22/

As always, we have guidelines on our website. There is on dedicated page https://paste.twilio.design/inclusive-design, and guidelines scattered throughout each foundation and component page. As an example, here's a really great example of [guidelines just on color](https://paste.twilio.design/foundations/colors#accessibility).

As for tools, we use [aXe](https://www.deque.com/axe/) which actually powers the Google Lighthouse audit. It'll catch the basics, but you can get a passing page and it still has accessibility issues. Every page should be tested, it's relatively straight forward to do with a good CI/CD pipeline.

The best approach would be Twilio having an Accessibility Conformance Report produced by a third party, based on a Voluntary Product Accessibility Template (VPAT).
